packages/typespec-azure-resource-manager/test/rules/missing-x-ms-identifiers.test.ts

```diff
@@ -130,4 +130,29 @@ describe("typespec-azure-core: no-enum rule", () => {
       )
       .toBeValid();
   });
+
+  it("allow array of x-ms-identifiers", async () => {
+    await tester
+      .expect(
+        `
+        model Pet {
+          @OpenAPI.extension("x-ms-identifiers", ["food/brand/name"])
```

Comment:
The doc linked in the issue mentioned that those should start with / in the example.

Not really sure if that's accurate but maybe we should support it or confirm there isn't use case of one or the other